CONTRIBUTING.md

```diff
@@ -109,16 +109,31 @@ Each lesson has two maintainers who review issues and pull requests or encourage
 The maintainers are community volunteers and have final say over what gets merged into the lesson.
 To use the web interface for contributing to a lesson:
 
-1.  Fork the originating repository to your GitHub profile.
-2.  Within your version of the forked repository, move to the `gh-pages` branch and
+1. Fork the originating repository to your GitHub profile.
```

Comment:
@chuckwondo can you fix these bullet points so they are numbered again?

Reply:
oh cool! I didn't know that. good as is then.

Reply:
Cool. Anything else?

Reply:
LGTM!